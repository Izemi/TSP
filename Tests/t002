#!/bin/bash
# Different 6 cities

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


$BIN/run -stderr=/dev/null ./TSP $WHERE/wa_6.dat -given SEA PSC ALW GEG BLI PAE < /dev/null
