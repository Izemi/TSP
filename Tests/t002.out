-given    :      1067.69 SEA PSC ALW GEG BLI PAE SEA
